Dockerfile

```diff
@@ -45,7 +45,25 @@ COPY --from=frontend /home/app/spark-${SPARK_VERSION}-bin-hadoop${HADOOP_VERSION
 
 COPY k8s/ ./k8s/
 
+ARG spark_uid=10000
+ARG spark_gid=10001
+RUN groupadd -g ${spark_gid} spark && useradd spark -u ${spark_uid} -g ${spark_gid} -m -s /bin/bash
+RUN mkdir -p /home/db /tmp/spark-events /tmp/staging /tmp/s3a ${SPARK_HOME}/workdir && \
```

Comment:
Why do we need to create all these `/home/db /tmp/spark-events /tmp/staging /tmp/s3a`? I do not think they are in use? And after that setting any permissions below seems unnecessary.

Reply:
Yes you're right, those directories are only needed in the spark image.
I used the /home/db directory to do some test using H2 on file system using a PVC for persistence on reboot.
It is not needed either